src/main/java/emissary/core/IBaseDataObject.java

```diff
@@ -653,15 +653,39 @@ static enum MergePolicy {
      * 
```

Comment:
Whenever we add a `@Deprecated` annotation, we should let devs know the preferred alternative so they have a path towards addressing any new warnings they'll see.
```suggestion
     *  @deprecated See {@link #setHistory(TransformHistory)}
     * 
```